.ci/gdbstub-test.sh

```diff
@@ -21,7 +21,7 @@ if [ -z ${GDB} ]; then
     exit 1
 fi
 
-build/rv32emu -g build/puzzle.elf &
+build/rv32emu -g build/bin/riscv32/puzzle &
```

Comment:
It is boring to name the directory as `build/bin` since [binary blob](https://en.wikipedia.org/wiki/Binary_blob) locates in `build` directory. You can reconstruct as `build/riscv32` and `build/linux-x64`.